data/ecosystems/p/polkadot.toml

```diff
@@ -300,6 +301,7 @@ github_organizations = [
   "https://github.com/InkSmartContract",
   "https://github.com/ParityAsia",
   "https://github.com/patractlabs",
+  "https://github.com/Polkadot-Blockchain-Academy",
```

Comment:
you can remove this lines since the taxonomy operates like a graph and will upstream the subecosystem's repos/devs